client/packages/common/src/ui/forms/JsonForms/components/Date.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { rankWith, ControlProps, isDateControl } from '@jsonforms/core';
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import { FormLabel, Box } from '@mui/material';
+import { BaseDatePickerInput } from '@openmsupply-client/common';
+
+export const dateTester = rankWith(5, isDateControl);
```

Comment:
as a new user to this - what is a `tester`?
Can you add a comment to explain?
Could they be grouped in a testers file? Would that make it clearer? or is it helpful to have them next to the component?